airflow/contrib/operators/aws_sqs_publish_operator.py

```diff
@@ -16,29 +15,14 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
+"""Publish message to SQS queue"""
 from airflow.models import BaseOperator
 from airflow.utils.decorators import apply_defaults
 from airflow.contrib.hooks.aws_sqs_hook import SQSHook
 
 
 class SQSPublishOperator(BaseOperator):
-    """
-    Publish message to a SQS queue.
-
-    :param sqs_queue: The SQS queue url (templated)
-    :type sqs_queue: str
-    :param message_content: The message content (templated)
-    :type message_content: str
-    :param message_attributes: additional attributes for the message (default: None)
-        For details of the attributes parameter see :py:meth:`botocore.client.SQS.send_message`
-    :type message_attributes: dict
-    :param delay_seconds: message delay (templated) (default: 1 second)
-    :type delay_seconds: int
-    :param aws_conn_id: AWS connection id (default: aws_default)
-    :type aws_conn_id: str
-    """
-
+    """Operator to publish message to AWS SQS queue """
```

Comment:
We should have docstring only in one place - constructor or class. Otherwise the documentation will not display correctly.

Reply:
Right. Fixed few other places as well.